src/main/kotlin/uk/gov/justice/digital/hmpps/visits/prison/config/resource/ExampleResource.kt

```diff
@@ -8,26 +8,25 @@ import io.swagger.v3.oas.annotations.security.SecurityRequirement
 import io.swagger.v3.oas.annotations.tags.Tag
 import org.springframework.security.access.prepost.PreAuthorize
 import org.springframework.web.bind.annotation.GetMapping
-import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
-import uk.gov.justice.digital.hmpps.visitsprisonconfigservice.service.ExampleApiService
-import uk.gov.justice.hmpps.kotlin.common.ErrorResponse
+import uk.gov.justice.digital.hmpps.visits.prison.config.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.visits.prison.config.service.ExampleApiService
```

Comment:
In this class and throughout, look for any TODO / temporary comments which we should be removing. 

E.g. line 17, 18 and 20 comments can be removed

Reply:
need to remove the whole class. Will do it on the next PR